test/mocks/renderedTemplate.ts

```diff
@@ -60,3 +60,13 @@ along with a new line to test the preservation of indentation
 - <small>📅 Highlight modified on:: 2024-03-11 03:04:53 PM -04:00</small>
 
 `;
+
+export const renderedWrappedTextBlockTemplateMock = `----
+
+> [!QUOTE]
+>  Chapter 1 introduces the terminology and approach
```

Comment:
Please update the test to use the proper highlight from the `aggregateWrappedTextBlockData.ts`